Barotrauma/BarotraumaShared/Source/Items/Item.cs

```diff
@@ -777,11 +777,13 @@ public AttackResult AddDamage(Character attacker, Vector2 worldPosition, Attack
 
         private bool IsInWater()
         {
+            if (parentInventory != null && parentInventory.Owner != null)
+                FindHull();
```

Comment:
The update method keeps calling FindHull as long as the item is moving, so why is this necessary? Now all items, including immovable ones without a body, keep calling it constantly (twice per frame if moving).

Reply:
Yeah this might not be neccessary at all now that I think about it.
The only reason I assumed this was neccessary is if it used incorrect hull, however I'm not sure if that's even the case.

Reply:
Disregard my previous comment actually, I didn't notice it's only done for items in an inventory. 😅 In any case I don't think this is an appropriate place for the hull checks, IMO it would make more sense to ensure the hull is up-to-date in the Update function.